nestkernel/synaptic_element.cpp

```diff
@@ -121,10 +121,10 @@ nest::SynapticElement::set( const DictionaryDatum& d )
 
   // Store values
   if ( d->known( names::growth_rate ) )
-    growth_rate_ = getValue< double_t >( d, names::growth_rate );
-  else
-    updateValue< double_t >( d, names::growth_rate, growth_rate_ );
-  updateValue< double_t >( d, names::tau_vacant, new_tau_vacant );
+  {
+    growth_rate_ = getValue< double >( d, names::growth_rate );
+  }
```

Comment:
This code seems too complicated, the original code was correct:
```c++
updateValue< double >( d, names::growth_rate, growth_rate_ );
```
This will change the value of `growth_rate_` if `d` contains a `growth_rate` entry and leave it unchanged otherwise. `updateValue()` does not change the dictionary `d`, but it returns `true` if `growth_rate_` has been changed, `false` otherwise. This is useful in some cases where checks or changes are required if a member has been modified.